utils/dockerfile_generator.py

```diff
@@ -0,0 +1,185 @@
+                            baseImage = 'python:2-alpine'
+                            is_python = True
+                        elif '#!/usr/bin/env python3\n' in source_code:
+                            # If we're building every image, we should save disk space w/ alpine
+                            baseImage = 'python:3-alpine'
+                            is_python = True
+                        
+                        # TODO: Add more runtime shebangs! ex. go, rust, ruby
+
+                        # Default out to ubuntu
+                        else:
+                            baseImage = 'ubuntu'
+
+
+                if 'alpine' in baseImage:
+                    is_alpine = True
+
+                dockerfile_contents.append('FROM {}\n'.format(baseImage))
+
+
+                # Include alpine-specific dependencies
+                if is_alpine:
+
+                    # Using a set to prevent duplicate install operations
+                    alpine_dependencies = set()
+                    if is_python:
+                        requirements_path = analyzer / 'requirements.txt'
+                        if requirements_path.exists():
+                            requirements = requirements_path.open().read()
+                            
+                            if 'yara-python' in requirements:
+                                alpine_dependencies.add('gcc')
+                                alpine_dependencies.add('musl-dev')
+                            if 'python-magic' in requirements:
+                                alpine_dependencies.add('libmagic')
+                            if 'eml_parser' in requirements:
+                                alpine_dependencies.add('libmagic')
+                                alpine_dependencies.add('g++')
+                            if 'pyimpfuzzy' in requirements:
+                                alpine_dependencies.add('gcc')
+                                alpine_dependencies.add('musl-dev')
+                            if 'autofocus-client-library' in requirements:
+                                alpine_dependencies.add('gcc')
+                                alpine_dependencies.add('musl-dev')
+                            if 'domaintools_api' in requirements:
+                                alpine_dependencies.add('gcc')
+                                alpine_dependencies.add('musl-dev')
+
+                            # One of the requirements is a git repository-- include git
+                            if 'git+https' in requirements:
+                                alpine_dependencies.add('git')
+                    else:
+                        # TODO: Add more language support here
+                        pass
+
+
+                    if alpine_dependencies:
+                        dockerfile_contents.append('RUN apk add --no-cache {}\n'.format(' '.join(sorted(alpine_dependencies))))
+
+
+                labels = dict()
```

Comment:
Here down is the `LABEL` metadata tags. Analyzers with multiple authors will have an `authors` label and a CSV of the contributors' names in alphabetical order.